lib/classes/ConfigSchemaHandler/index.js

```diff
@@ -99,6 +101,8 @@ class ConfigSchemaHandler {
     }
 
     const ajv = new Ajv({ allErrors: true, coerceTypes: 'array', verbose: true });
+    ajv.addKeyword('deprecation', { validate: () => false });
```

Comment:
How that behaves if there are other validation errors on a deprecated property. Will they also surface?